src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -740,10 +740,10 @@ window.qBittorrent.DynamicTable = (function() {
                     const tr = new Element("tr");
                     // set tabindex so element receives keydown events
                     // more info: https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event
-                    tr.setProperty("tabindex", "-1");
```

Comment:
This change broke https://github.com/qbittorrent/qBittorrent/pull/15186. It should be a real attribute to work.